src/dvclive/live.py

```diff
@@ -299,6 +301,19 @@ def log_param(
         """Saves the given parameter value to yaml"""
         self.log_params({name: val})
 
+    def log_artifact(
+        self,
+        path: str,
+    ):
+        """Saves the given parameter value to yaml"""
+        if self._dvc_repo is not None and not self._inside_dvc_exp:
+            dvc_file = self._dvc_repo.add(path)[0].addressing
```

Comment:
Should we check if it already exists as an output in either `dvc.yaml` or `.dvc`?

Reply:
I think we can wrap the exception here.